webapp/secrets.py

```diff
@@ -15,8 +17,17 @@ def get_secret(secret_name, region_name="sa-east-1"):
             SecretId=secret_name
         )
     except ClientError as e:
+    """    
         # For a list of exceptions thrown, see
         # https://docs.aws.amazon.com/secretsmanager/latest/apireference/API_GetSecretValue.html
-        raise e
 
-    return json.loads(json.loads(get_secret_value_response['SecretString'])['credentials'])
\ No newline at end of file
+    try:
+        with open(secrets_path, 'r') as f:
+            secrets = json.load(f)
+            return secrets[secret_name]
+            print(secrets)
+    except Exception as e_:
+        print(e_)
+        #raise e_ from e
+
+    #return json.loads(json.loads(get_secret_value_response['SecretString'])['credentials'])
```

Comment:
Retirar os prints e colocar o return de volta senão vai quebrar produção:
```suggestion
    except Exception as e_:
        raise e_ from e

    return json.loads(json.loads(get_secret_value_response['SecretString'])['credentials'])
```